File: DebtAngular/Models/Member.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace DebtAngular.Models
{
    public class Member : CommonModel
    {
        public string Name { get; set; }
        public double Deposit { get; set; }
        public double Debt { get; set; }

        [ForeignKey("Task")]
        public Guid TaskId { get; set; }
        public virtual Task Task { get; set; }
    }
}

File: DebtAngular/ClientApp/src/app/_models/member.ts
import { DoBootstrap } from "@angular/core";

export class Member {
    id:string
    name:string;
    deposit:number;
    debt:number;
    taskId:string;
    constructor(taskId:string){
        this.id ='00000000-0000-0000-0000-000000000000';
        this.name='';
        this.deposit=0;
        this.debt=0;
        this.taskId = taskId;
    }
}

